R/stats_wrapper.R

```diff
@@ -199,7 +199,14 @@ do_cmdscale_ <- function(df,
         mat <- rbind(mat, matrix(0, nrow=1, ncol=ncol(mat), dimnames = list(diffrow, NULL)))
       }
     }
+
+    if (ncol(mat) <= k) {
+      # this causes an error in cmdscale, so should be validated
+      stop("Number of unique values in pairs should be more than the number of dimensions")
```

Comment:
I had wrong impression of value_col having something to do here.
"Number of unique pairs should be more than the number of dimensions"
might be less confusing.